src/core/mesh/qgsmeshrenderersettings.h

```diff
@@ -311,21 +311,17 @@ class CORE_EXPORT QgsMeshRendererVectorStreamlineSettings
  *
  * \note The API is considered EXPERIMENTAL and can be changed without a notice
  *
- * \since QGIS 3.2
+ * \since QGIS 3.12
```

Comment:
this was added in 3.2, undo your change please

Reply:
I don't know why I did this ...